docs/fsharp-core-notes.md

```diff
@@ -1,126 +1,118 @@
----
-layout: default
-title: Notes on FSharp.Core
-subtitle: This technical guide discusses the FSharp.Core library.  
----
-
 # Notes and Guidance on FSharp.Core
-{:.no_toc}
 
-This technical guide discusses the FSharp.Core library.  Please help improve this guide by editing it and submitting a pull-request.
+This technical guide discusses the FSharp.Core library.
+
+Reference documentation for FSharp.Core can be found here: https://fsharp.github.io/fsharp-core-docs/
 
 Much of the guidance below applies to any .NET library respecting binary compatibility.
 
-### FSharp.Core is binary compatible
+## FSharp.Core is binary compatible
 
-FSharp.Core is binary compatible across versions of the F# language. For example, FSharp.Core `5.0.0.0` (F# 5.0) is binary compatible with
-`4.7.0.0` (F# 4.7), `4.6.0.0` (F# 3.6), `4.4.0.0` (F# 4.0) , `4.4.1.0` (F# 4.1), , `4.4.3.0` (F# 4.1+)  and so on.
+FSharp.Core is binary compatible across versions of the F# language. For example, this means you can create a newer project with a newer FSharp.Core in an older codebase and things should generally "just work".
```

Comment:
FSharp.Core 4.0.0.0 and upwards is not Binary compatible with apps built with FSharp 2.0 and FSharp.Core 2.0.0.0.  Some types Tuple and Bigint for example moved to the Net framework.